src/Workspaces/CoreTest/Shared/Extensions/TelemetryExtensions/TelemetryExtensionTests.cs

```diff
@@ -5,10 +5,10 @@
 #nullable disable
 
 using System;
-using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.CodeAnalysis.Shared.Extensions;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Extensions
+namespace Microsoft.CodeAnalysis.UnitTests.Extensions
```

Comment:
This namespace change means the expected telemetryId being tested for has changed to `00000000-0000-0000-c4c5-914100000000`